linux_os/guide/system/software/updating/clean_components_post_updating/rule.yml

```diff
@@ -1,13 +1,17 @@
 documentation_complete: true
 
-prodtype: ol7,ol8,rhel7,rhel8,rhv4
```

Comment:
Remove this change from this pull request as it is already being covered in #6633 https://github.com/ComplianceAsCode/content/pull/6633/files?file-filters%5B%5D=.yml#diff-f0247c1c3e940c8b4c066c11094c9875a15cd3d0f0563929cf3a79f360c38349

Reply:
Actually, let's keep this one since it contains the newest change around the `platform` tag

```
{{% if product == 'sle12' %}}
platform: zypper
{{% else %}}
platform: yum
{{% endif %}}
```

Reply:
OK, will get it out of 6633, once this lands.